<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>
###############################################################################
#
#  EGSnrc user interface for egs_gui pegs page
#  Copyright (C) 2015 National Research Council Canada
#
#  This file is part of EGSnrc.
#
#  EGSnrc is free software: you can redistribute it and/or modify it under
#  the terms of the GNU Affero General Public License as published by the
#  Free Software Foundation, either version 3 of the License, or (at your
#  option) any later version.
#
#  EGSnrc is distributed in the hope that it will be useful, but WITHOUT ANY
#  WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
#  FOR A PARTICULAR PURPOSE.  See the GNU Affero General Public License for
#  more details.
#
#  You should have received a copy of the GNU Affero General Public License
#  along with EGSnrc. If not, see http://www.gnu.org/licenses/.
#
###############################################################################
#
#  Author:          Iwan Kawrakow, 2003
#
#  Contributors:    Ernesto Mainegra-Hing
#
###############################################################################
</comment>
 <class>EGS_PegsPage</class>
 <widget class="QWidget" name="EGS_PegsPage">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1886</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox6">
       <property name="minimumSize">
        <size>
         <width>261</width>
         <height>281</height>
        </size>
       </property>
       <property name="title">
        <string>Medium composition</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableWidget" name="composition_table">
          <property name="rowCount">
           <number>20</number>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <column>
           <property name="text">
            <string>Element</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Fraction by weight</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox11">
         <property name="title">
          <string>Medium name</string>
         </property>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLineEdit" name="medname_le"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QGroupBox" name="groupBox8">
           <property name="title">
            <string>Medium type</string>
           </property>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QComboBox" name="medtype_cbox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <item>
               <property name="text">
                <string>Element</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Compound</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Mixture</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="rho_group">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="title">
            <string>Mass density</string>
           </property>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QLineEdit" name="rho_le"/>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox2">
              <item>
               <property name="text">
                <string>g/cm**3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>kg/m**3</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox3">
         <property name="title">
          <string>Options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QCheckBox" name="dc_icru_check">
              <property name="text">
               <string>ICRU density correction</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="rad_icru_check">
              <property name="text">
               <string>ICRU radiative stopping power</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="rayleigh_check">
              <property name="text">
               <string>Include Rayleigh data</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QCheckBox" name="is_gas">
                <property name="text">
                 <string>Medium is a gas</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>9</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="gaspLabel">
                <property name="text">
                 <string>gas pressure</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="gaspEdit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="gaspUnits">
                <property name="text">
                 <string>atm</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="spacer5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>30</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="dc_group">
     <property name="title">
      <string>Density correction file</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLabel" name="textLabel5">
        <property name="text">
         <string>Look in</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="dc_where">
        <item>
         <property name="text">
          <string>HEN_HOUSE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EGS_HOME</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dc_file"/>
      </item>
      <item>
       <widget class="QPushButton" name="dc_button">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox5">
     <property name="title">
      <string>Energy range</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="textLabel1">
          <property name="text">
           <string>AE</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ae_le"/>
        </item>
        <item>
         <widget class="QComboBox" name="ae_units">
          <item>
           <property name="text">
            <string>keV</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MeV</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="textLabel2">
          <property name="text">
           <string>AP</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ap_le"/>
        </item>
        <item>
         <widget class="QComboBox" name="ap_units">
          <item>
           <property name="text">
            <string>keV</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MeV</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="textLabel3">
          <property name="text">
           <string>UE</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ue_le"/>
        </item>
        <item>
         <widget class="QComboBox" name="ue_units">
          <item>
           <property name="text">
            <string>keV</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MeV</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>GeV</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="textLabel4">
          <property name="text">
           <string>UP</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="up_le"/>
        </item>
        <item>
         <widget class="QComboBox" name="up_units">
          <item>
           <property name="text">
            <string>keV</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MeV</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>GeV</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox10">
     <property name="title">
      <string>PEGS Output</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QCheckBox" name="new_data_file">
          <property name="text">
           <string>Create new data file</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>85</width>
            <height>21</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="append_to_datafile">
          <property name="text">
           <string>Append to existing data file</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>84</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="textLabel6">
          <property name="text">
           <string>Output file</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ofile_le"/>
        </item>
        <item>
         <widget class="QPushButton" name="ofile_b">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer name="spacer4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>330</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="go_button">
       <property name="text">
        <string>&amp;Go</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="details_b">
       <property name="text">
        <string>&amp;Details&gt;&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>dc_icru_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>EGS_PegsPage</receiver>
   <slot>densityIcruChanged(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>medtype_cbox</sender>
   <signal>activated(QString)</signal>
   <receiver>EGS_PegsPage</receiver>
   <slot>medtypeChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dc_button</sender>
   <signal>clicked()</signal>
   <receiver>EGS_PegsPage</receiver>
   <slot>getDensityFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>details_b</sender>
   <signal>clicked()</signal>
   <receiver>EGS_PegsPage</receiver>
   <slot>showHideDetails()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>is_gas</sender>
   <signal>toggled(bool)</signal>
   <receiver>EGS_PegsPage</receiver>
   <slot>enable_gaspEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1032</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>942</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
